admin/cron/daily-search-indexes-dump.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu
+
+MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../../" && pwd)
+cd "$MB_SERVER_ROOT"
+
+# Only do this on the nominated days (0=Sun 6=Sat)
+if date +%w | grep -q [36]
```

Comment:
Maybe it should be 3=Wed 6=Sat instead? :)

Reply:
The comment gives the full range of week days returned by `date +%w` assuming `C` locale.
It differs from other locales/systems that may use `(0=Mon 6=Sun)` or `(1=Sun 7=Sat)`.

Reply:
I understood that, but why make people calculate what 3 is then? They don't need to know what Sunday is, just what the ones we use are :)

Reply:
People need to know the full range to set another value correctly, the ones we use are not enough, for example the full range could also be `(1=Mon 7=Sun)` (as with `date +%u`).
Rephrased with both used values and smallest allowed value in https://github.com/metabrainz/musicbrainz-server/pull/1407/commits/3f8bd723029f907af61c12904488b01f2aa5ea8e.